extensions/replication/management.js

```diff
@@ -67,11 +86,19 @@ function installReplicationConfiguration(bucketName, endpoint, workflows, cb) {
         },
     };
 
-    getS3Client(endpoint).putBucketReplication(params, err => {
-        logger.debug('replication configuration apply done', {
-            sourceBucket: bucketName, error: err });
-        return cb(err);
-    });
+    (async () => {
+        try {
+            const command = new PutBucketReplicationCommand(params);
+            await getS3Client(endpoint).send(command);
+            logger.debug('replication configuration apply done', {
+                sourceBucket: bucketName });
+            cb();
```

Comment:
do not invoke the continuation callback from within the try block.
Use `promise.then` or better yet `utils.callbackify`